.github/workflows/rspec-system.yml

```diff
@@ -0,0 +1,93 @@
+name: rspec-system
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
+      - 'bin/*'
+
+jobs:
+  rspec-system:
+
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: postgres:12.3
+        env:
+          POSTGRES_PASSWORD: password
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    # https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix
+    strategy:
+      fail-fast: false
+      matrix:
+        # Set N number of parallel jobs you want to run tests on.
+        # Use higher number if you have slow tests to split them on more parallel jobs.
+        # Remember to update ci_node_index below to 0..N-1
+        ci_node_total: [6]
+        # set N-1 indexes for parallel jobs
+        # When you run 2 parallel jobs then first job will have index 0, the second job will have index 1 etc
+        ci_node_index: [0, 1, 2, 3, 4, 5]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Install PostgreSQL client
+        run: |
+          sudo apt-get -yqq install libpq-dev
+      - name: Build App with asset compilation
+        env:
+          POSTGRES_HOST: localhost
+          DATABASE_HOST: localhost
+          PG_USERNAME: postgres
+          PG_PASSWORD: password
+          POSTGRES_HOST_AUTH_METHOD: trust
+          POSTGRES_PORT: 5432
+          RAILS_ENV: test
+        run: |
+          bundle exec skylight disable_dev_warning
+          yarn
+          bundle exec rake db:create
+          bundle exec rake db:schema:load
+          bundle exec rails assets:precompile
```

Comment:
By moving this to the workflow file, does this mean that running a single request/system spec locally won't work any more without manually running this? Is there a way to avoid that?

Reply:
Ahh good question, actually it should still work since we have a step to compile if the assets don't yet exist locally. Let me make sure that when we do run this we aren't compiling even though I'am changing it.

Reply:
Moved the logic for building assets to be before a system test so we know to build the assets.